base/reflection.jl

```diff
@@ -170,10 +170,11 @@ end
 
 tt_cons(t::ANY, tup::ANY) = (@_pure_meta; Tuple{t, (isa(tup, Type) ? tup.parameters : tup)...})
 
-code_lowered(f, t::ANY=Tuple) = map(m -> (m::Method).lambda_template, methods(f, t))
+code_lowered(f, t::ANY=Tuple) = map(m -> (m::Method).lambda_template, checked_methods(f, t))
```

Comment:
I believe this currently just returns an empty array instead of throwing an error.